yb-voyager/cmd/root.go

```diff
@@ -83,14 +82,11 @@ func init() {
 }
 
 func registerCommonGlobalFlags(cmd *cobra.Command) {
-	cmd.PersistentFlags().StringVarP(&logLevel, "log-level", "", "INFO",
-		"Logging levels: TRACE, DEBUG, INFO, WARN")
-
 	cmd.PersistentFlags().BoolVar(&source.VerboseMode, "verbose", false,
 		"enable verbose mode for the console output")
 
 	cmd.PersistentFlags().StringVarP(&exportDir, "export-dir", "e", "",
-		"export directory to keep all the dump files and metainfo")
+		"export directory is the workspace used to keep the exported schema, data, migration state and logs")
```

Comment:
directory which keeps the exported schema.....

Reply:
this also looks fine